app/src/main/kotlin/com/wire/android/migration/feature/MigrateClientsDataUseCase.kt

```diff
@@ -49,6 +49,10 @@ class MigrateClientsDataUseCase @Inject constructor(
     private val scalaUserDBProvider: ScalaUserDatabaseProvider,
     private val userDataStoreProvider: UserDataStoreProvider
 ) {
+
+    suspend operator fun invoke(userId: UserId, isFederated: Boolean): Either<CoreFailure, Unit> =
+        invoke(listOf(userId), isFederated).values.first()
+
     @Suppress("LoopWithTooManyJumpStatements", "ComplexMethod")
     suspend operator fun invoke(userIds: List<UserId>, isFederated: Boolean): Map<UserId, Either<CoreFailure, Unit>> {
 
```

Comment:
There are no usages of `invoke` with a list. We can simplify the code by removing it and the loop thingy.